Sources/Instructions/Helpers/Public/CoachMarkHelper.swift

```diff
@@ -61,9 +62,10 @@ public class CoachMarkHelper {
         withArrow arrow: Bool = true,
         arrowOrientation: CoachMarkArrowOrientation? = .top,
         hintText: String,
-        nextText: String? = nil
+        nextText: String? = nil,
+        nextLabelPosition: CoachMarkNextLabelPosition? = .none
     ) -> (bodyView: CoachMarkBodyDefaultView, arrowView: CoachMarkArrowDefaultView?) {
-        let coachMarkBodyView = CoachMarkBodyDefaultView(hintText: hintText, nextText: nextText)
+        let coachMarkBodyView = CoachMarkBodyDefaultView(hintText: hintText, nextText: nextText, nextLabelPosition: nextLabelPosition)
```

Comment:
Since there's a default value for the parameter, I think we can omit it.

```suggestion
        let coachMarkBodyView = CoachMarkBodyDefaultView(hintText: hintText, nextText: nextText)
```